report-summary-merged-prs

```diff
@@ -1068,6 +1068,7 @@ def find_check_headers(comparisons, architecture):
 
 def find_ubsan_logs(comparisons, arch):
     for c in comparisons:
+        if c['compared_tags'].find('UBSAN') < 2: continue
```

Comment:
what done `find()` return?

Reply:
I think you can just check of `UBSAN` in c['release_name']

Reply:
integer with how many times the substring is found into the searchstring

Reply:
sorry im thinking of count ill change it

Reply:
for me it returns the location where it find ubsan e.g
```
>python -c 'print("CMSSW_12_1_UBSAN_X_2021-09-22-1600-->CMSSW_12_1_X".find("UBSAN"))'
11
```

Reply:
I would suggest to just use `c['release_name']`

Reply:
yes sorry it returns the position of the first hit , should be count not find